### PR TITLE
V3.0.0

### PR DESCRIPTION
**Pull request by manast**
This is the PR that will ultimately become v3.0.0.
Comments and feedback welcomed.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -1,13 +0,0 @@
-
-License
```
Comment (bradvogel) [inline LICENSE.md]: accidental deletion?

File: support/logo.svg [hunk 2]
```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bradvogel) [inline support/logo.svg]: accidental removal

Comment (bradvogel) [inline support/logo.svg]: along with logo.png and logo.sketch above

File: README.md [hunk 3]
```diff
@@ -1,132 +1,85 @@
+Bull Job Manager
+================
 
-<div align="center">
```
Comment (bradvogel) [inline README.md]: Would be nice to keep the new formatting

File: REFERENCE.md [hunk 4]
```diff
@@ -1,459 +0,0 @@
-
```
Comment (bradvogel) [inline REFERENCE.md]: would be nice to keep this separate file, just update with docs from the 3.0 branch

**Comment by bradvogel** [comment]
Anything I can help with to get this merged?

**Comment by manast** [comment]
I have moved some of tv3.0.0 issues to v3.1.0, so there are only 3 left: https://github.com/OptimalBits/bull/milestone/4
I have started to work on naming the workers (including listing workers, etc). The other two are more of "defensive" issues, since as far as I know they do not resolve any known bug.

**Comment by bradvogel** [comment]
Awesome! Happy to review PRs. Excited for 3.0!

**Comment by bradvogel** [comment]
once you merge https://github.com/OptimalBits/bull/pull/537 into this - how about merging this into master and pushing an RC version? I'll start testing it in one of our prod environments.

**Pull request merged**